plugins/package-managers/node/src/main/kotlin/yarn2/Yarn2.kt

```diff
@@ -107,13 +126,14 @@ class Yarn2(override val descriptor: PluginDescriptor = Yarn2Factory.descriptor,
     ): List<ProjectAnalyzerResult> {
         val workingDir = definitionFile.parentFile
         installDependencies(workingDir)
+        moduleInfoResolver.workingDir = workingDir
```

Comment:
Nit: All other PMs do this assignment before installing dependencies. So maybe align this, and swap lines 128 and 129?